cmd/server.go

```diff
@@ -0,0 +1,33 @@
+package cmd
+
+import (
+	"errors"
+
+	"github.com/gpf-org/gpf/server"
+	"github.com/spf13/cobra"
+)
+
+var options = &server.ServerOptions{}
+
+var ServerCmd = &cobra.Command{
+	Use:   "server",
+	Short: "A high performance gpf server",
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if options.Token == "" {
+			return errors.New("missing required token flag")
+		}
+		return nil
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		s := server.Server{}
+		s.Start(options)
+	},
+}
+
+func init() {
+	ServerCmd.PersistentFlags().StringVarP(&options.Token, "token", "t", "", "Private token")
```

Comment:
At the moment both Token and BaseURL are provided as global flags however due to the fact that the logic will be running inside the server, the client part will be just http requests to the server, based on this the Token and BaseURL global flags will no longer be needed